gcpkms/mock_kms_server_test.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2019 Google LLC
```

Comment:
@ajvb copyright around this can not be avoided, it is copied from the tests of the SDK (with some changes to drop deprecated dependencies).